obs-browser-plugin.cpp

```diff
@@ -390,7 +392,10 @@ void RegisterBrowserSource()
 	};
 	info.show = [] (void *data)
 	{
-		static_cast<BrowserSource *>(data)->SetShowing(true);
+		BrowserSource *bs = static_cast<BrowserSource *>(data);
+		if (bs->estart_on_visible)
```

Comment:
I haven't tested this, but this looks like a typo to me.
Shouldn't this be `restart_on_visible`, not `estart_on_visible` ?

Reply:
Indeed ...
I'll fix this